src/QMCWaveFunctions/tests/test_einset_spinor.cpp

```diff
@@ -97,7 +97,7 @@ TEST_CASE("Einspline SpinorSet from HDF", "[wavefunction]")
 
 
   const char* particles = "<tmp> \
-   <sposet_builder name=\"A\" type=\"einspline\" href=\"o2_45deg_spins.pwscf.h5\" tilematrix=\"1 0 0 0 1 0 0 0 1\" twistnum=\"0\" source=\"ion\" size=\"3\" precision=\"float\"> \
```

Comment:
How  much does the memory increase? mesh 4 will be 64 times larger.

Reply:
Looks like the whole h5 file is 1M.

Reply:
When I was first making the test, I had to crank up the mesh factor to get the splines to exactly agree with the raw PW values and I just left it there. As far as memory is concerned, when running the test the up and down components each print `MEMORY 43 MB allocated...`. So if this is too high for a test, I can reduce the meshfactor and will therefore change the tolerances for the checks.

Reply:
43MB is not a concern

Reply:
The discussion here is about increasing the size of the git repo permanently.

Reply:
It is good that the large file was spotted. If there is a chance to make it smaller we should take advantage. however I think 1MB is fair provided there are not plans to keep updating it.

Newly developed or improved effective core potentials have been made available via the pseudopotentiallibrary.org website.

@camelto2 Do you think that this is a good file that will be stable for a long time? I assume so but if there is another system that you think might be more appropriate we should update.

Reply:
I don't see a reason why this file would need to change at all in the future. It is just the output from a QE noncollinear calculation, so it should be stable.

With that said, since I now have an independent tool to directly get the reference values from plane waves, I can run whatever system with a smaller PW cutoff and get a smaller hdf5 file if that is desired.

We just chose this system because we used it to benchmark the noncollinear wave functions against some collinear calculations in the past.

Reply:
@prckent I got confused by your message. the h5 in the repo is 924K. I saw an change of mesh factor in the test and thus I was wondering if memory needs for testing increases or not. 43M memory in a unit test is not an issue.

Reply:
I thought this was a new file. In any case, good to have the discussion.